report/docker_run.sh

```diff
@@ -19,13 +19,16 @@
 ##
 ##   benchmark_set determines the set of benchmarks used for the experiment.
 ##     The value should be the name of `benchmark-sets` directory.
-##     Default: comp_benchmarks
+##     Default: comparison
```

Comment:
@erfanio could you please also apply this change to your PR?
Thanks!

Reply:
I created #51